docs/specs/restore.yml

```diff
@@ -104,7 +104,7 @@ inputs:
     extend: https://this-url-does-not-exit.com/this-is-a-bad-url
 outputs:
   build.log: |
-    ["error","download-failed","Download 'https://this-url-does-not-exit.com/this-is-a-bad-url' failed: *"]
+    ["error","download-failed","System: Download failed for file 'https://this-url-does-not-exit.com/this-is-a-bad-url'. Try closing and reopening the PR. If you get this Error again, file an issue."]
```

Comment:
We should have a separate category indicating whether this is a system error, with that, we may remove `System: ` prefix.

Reply:
Removed. The category seems not working for now, build report always shows `Unspecific` type for docfx log codes.